examples/juniper/ncptx/ncptx-cptx.pb.txt

```diff
@@ -0,0 +1,154 @@
+name: "ncptx-cptx-nsoni"
+nodes: {
+    name: "ncptx"
+    vendor: JUNIPER
+    model: "ncptx"
+    os: "evo"
+    config: {
+      config_path: "/home/evo/configdisk/"
+      config_file: "juniper.conf"
+      file: "juniper.config"
+      image: "ncptx:latest"
+      cert: {
+            self_signed: {
+                cert_name: "grpc-server-cert",
+                key_name: "N/A",
+                key_size: 4096,
+            }
+        }
+    }
+    interfaces: {
+        key: "eth5"
+        value: {
+          name: "et-0/0/0"
+        }
+    }
+    interfaces: {
+        key: "eth6"
+        value: {
+          name: "et-0/0/1"
+        }
+    }
+    services: {
+        key: 22
+        value: {
+            name: "ssh"
+            inside: 22
+        }
+    }
+    services:{
+        key: 9337
+        value: {
+            name: "gnoi"
+            outside: 9337
+            inside: 32767
+        }
+    }
+    services:{
+        key: 9339
+        value: {
+            name: "gnmi"
+            outside: 9339
+            inside: 32767
+        }
+    }
+    services:{
+        key: 9340
+        value: {
+            name: "gribi"
+            outside: 9340
+            inside: 32767
+        }
+    }
+    services:{
```

Comment:
you can add p4rt to the node default services and remove them from here to reduce verbosity